### PR TITLE
Michael branch

### PR DESCRIPTION
**Pull request by MichaelKorost**
hey, big boss

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -67,5 +67,6 @@ <h1>Games</h1>
 <script src="./Js/index.js"></script>
 
 </body>
-
+<!-- hello, how are you? -->
+<!-- hello, how are you? -->
```
Comment (talbest) [inline index.html:71]: lol what a NOOOBB !!!@#!@#!@#

File: index.html [hunk 2]
```diff
@@ -67,5 +67,6 @@ <h1>Games</h1>
 <script src="./Js/index.js"></script>
 
 </body>
-
```
Comment (talbest) [inline index.html:70]: what a kid

File: index.html [hunk 3]
```diff
@@ -67,5 +67,6 @@ <h1>Games</h1>
 <script src="./Js/index.js"></script>
```
Comment (talbest) [inline index.html:68]: lolllll

Comment (MichaelKorost) [reply index.html:68]: thank you, very helpfull!

**Pull request merged**